### PR TITLE
Version of server download

### PR DESCRIPTION
**Pull request by anthonydillon**
## Done
Added the three versions of the sign up in the download server thank you page.

## QA
- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Change `default='true'` to the other version and check they match the designs in the issues and the [copy doc](https://docs.google.com/document/d/1S5zY7HN_Q8eQWch0mCEfLICfj8pW1bct9_wGArCNIqc/edit#)

## Issue / Card
Fixes: https://github.com/ubuntudesign/web-squad/issues/831
Fixes: https://github.com/ubuntudesign/web-squad/issues/834
Fixes: https://github.com/ubuntudesign/web-squad/issues/833

## Screenshots
screenshot_2018-09-21 thanks for downloading ubuntu server ubuntu 2
screenshot_2018-09-21 thanks for downloading ubuntu server ubuntu 1
screenshot_2018-09-21 thanks for downloading ubuntu server ubuntu

### CODE REVIEW

**Comment by caldav** [comment]
@anthonydillon to make testing straightforward, could we try switching between them with a URL parameter? Unless you had something in mind.

Currently, the URL format is: `https://www.ubuntu.com/download/server/thank-you?country=FR&version=18.04.1&architecture=amd64`

What about adding something (very literal), like `&suggested-read=1`, `2` and `3`?

**Comment by frankiegranato** [comment]
Hi @anthonydillon. This looks good to me. The form fields are all present and correct.  I know @anasereijo has some feedback for you about the H1 font weight, and she'll also need to give you an amended version of the illustration, but aside from that its a +1 from me.

**Comment by anasereijo** [comment]
@anthonydillon 

- can you make `h1 thin` on all versions?

Version A and B
- something went wrong with the MAAS illustration: please replace with [this one](https://assets.ubuntu.com/v1/5e6b2dce-MAAS-ebook-web.svg)
- seems like there's an empty column between the text and form (I only asked that for version C), can you please amend it and make it look like my designs? it was originally like that so we want to keep it as it was on both versions


Version C
- the ebook section title should be in 2 lines and as part of the right side section (check my design)
- the form should move up and top aligned with the title
- the T&cs below the CTA should be 14px 
45872032-46c3a500-bd8f-11e8-9342-9749bec7fc6f

**Comment by anthonydillon** [comment]
@anasereijo @caldav Done all the changes:
screenshot_2018-09-26 thanks for downloading ubuntu server ubuntu
screenshot_2018-09-26 thanks for downloading ubuntu server ubuntu 1
screenshot_2018-09-26 thanks for downloading ubuntu server ubuntu 2

File: templates/download/server/thank-you.html [hunk 1]
```diff
@@ -28,13 +28,15 @@ <h1>Thank you for downloading Ubuntu Server</h1>
...
     <div class="col-4 u-vertically-center u-align--center">
-      <img src="{{ ASSET_SERVER_URL }}52d53696-picto-thankyou-midaubergine.svg"
+      <img src="{{ ASSET_SERVER_URL }}cbae9a60-thank_you_orange_cmyk.svg"
         alt="smile" width="200" height="200" />
     </div>
   </div>
 </div>
 
-{% include "download/shared/_get_ebook.html"  with return_url="https://www.ubuntu.com/server/thank-you" %}
+{% include "download/shared/_get_ebook_v1.html" with return_url="https://www.ubuntu.com/server/thank-you" default="true" %}
```
Comment (caldav) [inline templates/download/server/thank-you.html]: Should be /download/server/thank-you

File: templates/download/server/thank-you.html [hunk 2]
```diff
@@ -28,13 +28,15 @@ <h1>Thank you for downloading Ubuntu Server</h1>
...
+{% include "download/shared/_get_ebook_v2.html" with return_url="https://www.ubuntu.com/server/thank-you" %}
```
Comment (caldav) [inline templates/download/server/thank-you.html]: Should be /download/server/thank-you

File: templates/download/server/thank-you.html [hunk 3]
```diff
@@ -28,13 +28,15 @@ <h1>Thank you for downloading Ubuntu Server</h1>
...
+{% include "download/shared/_get_ebook_v3.html" with return_url="https://www.ubuntu.com/server/thank-you" %}
```
Comment (caldav) [inline templates/download/server/thank-you.html]: Should be /download/server/thank-you

File: templates/download/shared/_get_ebook_v2.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+    }
+
+    /**
+     * After submit has happened
+     * start download and send the user to the instructions page
+     */
+    afterSubmit = function() {
+      // Now start the download
+      downloadFrame = document.createElement("iframe");
+      downloadFrame.setAttribute("src", "{{ ASSET_SERVER_URL }}e922aac4-OpenStack-made-easy_eBook_11.17.zip");
```
Comment (caldav) [inline templates/download/shared/_get_ebook_v2.html]: Wrong ebook here, should be: {{ ASSET_SERVER_URL }}05f610df-eBook_MAAS.pdf.zip

File: templates/download/shared/_get_ebook_v3.html [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+    }
+
+    /**
+     * After submit has happened
+     * start download and send the user to the instructions page
+     */
+    afterSubmit = function() {
+      // Now start the download
+      downloadFrame = document.createElement("iframe");
+      downloadFrame.setAttribute("src", "{{ ASSET_SERVER_URL }}e922aac4-OpenStack-made-easy_eBook_11.17.zip");
```
Comment (caldav) [inline templates/download/shared/_get_ebook_v3.html]: Wrong ebook here, should be: {{ ASSET_SERVER_URL }}05f610df-eBook_MAAS.pdf.zip

**Comment by WillMoggridge** [comment]
Demo URL has changed:
http://www.ubuntu.com-canonical-websites-pr-4076.run.demo.haus/

**Comment by anthonydillon** [comment]
@WillMoggridge thanks Willbot

**Comment by anasereijo** [comment]
@anthonydillon looks great 👍

**Pull request merged**